uint32 CAN_ID
uint8 command_type
float32 value
char value_type